bees/telegrambee/telegrambeefactory.go

```diff
@@ -99,6 +99,11 @@ func (factory *TelegramBeeFactory) Events() []bees.EventDescriptor {
 					Description: "User ID sending the message",
 					Type:        "string",
 				},
+				{
+					Name:        "timestamp",
+					Description: "Timestamp",
+					Type:        "time.Time",
```

Comment:
I'd keep this simple and still call the type `timestamp`. Its internal Go type is separate to that type definition here, which we mostly use for the admin interface.

Reply:
Ahh gotcha, wasn't sure of that bit.